packages/webapp/public/locales/en/translation.json

```diff
@@ -1,4 +1,21 @@
 {
+  "ADD_ANIMAL": {
+    "ANIMALS_TOTAL": "{{count}} animal total",
+    "ANIMALS_TOTAL_one": "{{count}} animal total",
+    "ANIMALS_TOTAL_other": "{{count}} animals total",
+    "ANIMALS_UNSPECIFIED": "{{count}} unspecified",
+    "ANIMALS_UNSPECIFIED_one": "{{count}} unspecified",
+    "ANIMALS_UNSPECIFIED_other": "{{count}} unspecified",
+    "SELECT_SEXES": "Select sexes",
+    "SEX_DETAIL": "Sex detail",
+    "SEX_DETAIL_ERROR": "Maximum animal count: {{count}}",
+    "SEX_DETAIL_ERROR_one": "Maximum animal count: {{count}}",
+    "SEX_DETAIL_ERROR_other": "Maximum animal count: {{count}}",
+    "SEX_DETAIL_ERROR_POPOVER": "You cannot have more than {{count}} animals",
```

Comment:
If you have included `_one` and `other` I don't think the un-suffixed version is needed.

Reply:
If you leave out the un-suffixed version then the next time someone runs the script, wouldn't all the un-suffixed versions be auto added and then they have to delete them?

Reply:
The script is generating those un-suffixed versions for you? I can delete those and re-run the script and they aren't re-generated...

Reply:
Oh nvm it's not generating them, I don't know why I thought they would be generated

Reply:
I removed the un-suffixed keys, although for es the `SELECTED` key was already translated so I duplicated it for _one, _many, _other.

Reply:
> although for es the un-suffixed SELECTED key was already translated so I duplicated it for _one, _many, _other.

Okay 👍  That's a plural form though, so I it won't work for `_one`; I'll leave a suggestion. It's also totally fine to leave it all MISSING (even if it means deleting a previously translated key) to let a native speaker do the pluralization... however this is an easy one so I'll put it in.

And it looks like other the un-suffixed forms (the ones under `ADD_ANIMAL`) need to still be removed from ES/FR/PT?

Reply:
Ok thanks for filling the es translation I had no idea about that one. And I removed the remaining un-suffixed ones from es/fr/pt - hopefully I didn't miss anything this time.